File: interview_prep/Fundamentals/sigma.js
// Implement a function sigma(num) that, given a number, returns the sum of all positive integers from 1 up to number (inclusive).

function sigma(num) {
    let sum = 0
    for (let i = 0; i <= num; i++) {
        sum += i
    }
    return sum
}

console.log(sigma(5))

// es6

const sigma2 = num => {
    return (num * (num+1)) /2
}

console.log(sigma2(5))

File: interview_prep/Fundamentals/sigma.py
# Implement a function sigma(num) that, given a number, returns the sum of all positive integers from 1 up to number (inclusive).

def sigma(num):
    sum = 0
    for i in range(0, num + 1, 1):
        sum += i
    return sum
        
print(sigma(5))